includes/class-assets.php

```diff
@@ -485,4 +488,35 @@ private static function is_jquery_ui_css_loaded() {
 		}
 		return false;
 	}
+
+	/**
+	 * Handles Floating Links' scripts and styles enqueueing.
+	 *
+	 * @since x.x
+	 *
+	 * @param string $plugin_url URL of the plugin.
+	 * @param string $version Current version of the plugin.
+	 * @return void
+	 */
+	private static function enqueue_floating_links( $plugin_url, $version ) {
+		if ( ! $plugin_url || ! $version ) {
+			// If any required parameters are missing, exit early.
+			return;
+		}
+
+		// Enqueue the Floating Links styles.
+		wp_enqueue_style( 's11-floating-links', $plugin_url . '/css/packages/s11-floating-links.css', array(), $version );
+
+		// Enqueue the Floating Links scripts.
+		wp_enqueue_script( 's11-floating-links-notifications', $plugin_url . '/js/packages/floating-links/s11-floating-links-notifications.js', array(), $version, true );
+		wp_enqueue_script( 's11-floating-links', $plugin_url . '/js/packages/floating-links/s11-floating-links.js', array(), $version, true );
```

Comment:
Of course, I'll incorporate your input. My goal is to make updates easier to manage across multiple plugins. Since we don't have notifications in FF, I've decided to create a separate file that we can easily copy and paste if needed. The idea is to only change the `config.js` file and keep `s11-floating-links.js` and `s11-floating-links-notifications.js` the same across all plugins, maintaining consistent functionality.

Reply:
We can ignore this for now because I've researched how we could combine them. Implementing this would require some setup in our plugin, which could be time-consuming. Below are the structures for your consideration. Please let me know what your thoughts are. Do you think we should proceed with this?

1. **Run the following commands to install `grunt-babel`, `@babel/core`, and `@babel/preset-env`:**

    ```bash
    npm install grunt-babel @babel/core @babel/preset-env --save-dev
    ```

2. **Update your `Gruntfile.js` to include a new Babel task. You'll add the following section:**

    ```javascript
    // Add these lines at the top of your Gruntfile.js
    const babel = require('grunt-babel');
    
    // ...existing code
    
    module.exports = function( grunt ) {
        // ...existing config
    
        // Add this inside the 'grunt.initConfig' function
        grunt.config.set('babel', {
            options: {
                sourceMap: true,
                presets: ['@babel/preset-env']
            },
            dist: {
                files: [
                    {
                        expand: true,
                        cwd: 'src/js/',  // replace with your source JavaScript files path
                        src: ['*.js'],
                        dest: 'dist/js/',  // replace with your transpiled files destination folder
                        ext: '.js'
                    }
                ]
            }
        });
    
        // ...existing tasks
    
        // Load grunt-babel
        grunt.loadNpmTasks('grunt-babel');
    
        // ...existing code
    };
    ```

3. **Add `babel` to your task lists wherever you'd like it to run. For example:**

    ```javascript
    // Inside your 'minify' task or any other task where you want to run Babel
    if ('undefined' !== typeof grunt.config.get('babel')) {
        grunt.task.run('babel');
    }
    ```

Reply:
@shervElmi I don't think this would take much time, but I don't think we need to make it a priority right now. We can skip it since this is only on BD pages in the admin.